lua-mode.el

```diff
@@ -440,7 +440,8 @@ traceback location."
             "type" "unpack" "xpcall" "self"
             ("bit32" . ("arshift" "band" "bnot" "bor" "btest" "bxor" "extract"
                         "lrotate" "lshift" "replace" "rrotate" "rshift"))
-            ("coroutine" . ("create" "resume" "running" "status" "wrap" "yield"))
+            ("coroutine" . ("create" "isyeldable" "resume" "running" "status"
```

Comment:
Oops, yes, obviously. I'll amend that immediately.

Le 18 mai 2015 à 18:19 CEST, dennis again a écrit :
>> @@ -440,7 +440,8 @@ traceback location."
>>              "type" "unpack" "xpcall" "self"
>>              ("bit32" . ("arshift" "band" "bnot" "bor" "btest" "bxor" "extract"
>>                          "lrotate" "lshift" "replace" "rrotate" "rshift"))
>> -            ("coroutine" . ("create" "resume" "running" "status" "wrap" "yield"))
>> +            ("coroutine" . ("create" "isyeldable" "resume" "running" "status"
>
> did you mean `isyieldable`?
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/immerrr/lua-mode/pull/90/files#r30522908

-- 
Thomas/Schnouki